src/containers/categoryFilter.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CategoryFilter = ({ filter }) => {
+  const categories = ['All', 'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  const [category, setCategory] = React.useState(categories[0]);
+  return (
+    <div>
+      <select
+        name="category"
+        onChange={e => setCategory(e.target.value)}
+      >
+        {categories.map(category => (
+          <option key={category}>
+            {category}
+          </option>
+        ))}
+      </select>
+      <button type="button" onClick={() => filter(category)}> Filter </button>
```

Comment:
- [ ] Why not make this process automatic without having to press the button? You can run the `handleFilterChange` right after the `select` tag is changed